backport/backport.ts

```diff
@@ -166,11 +166,13 @@ const getFailedBackportCommentBody = ({
 		'# Fetch latest updates from GitHub',
 		'git fetch',
 		'# Create a new working tree',
-		`git worktree add ${worktreePath} ${base}`,
+		`git worktree add ${worktreePath} origin/${base}`,
 		'# Navigate to the new working tree',
 		`cd ${worktreePath}`,
 		'# Create a new branch',
 		`git switch --create ${head}`,
+		'# Install dependencies for pre-commit hooks',
+		`yarn install --frozen-lockfile`,
```

Comment:
dont think we need this, this is part of the devenv setup

Reply:
@torkelo It's necessary because you enter a new worktree, where the dependencies aren't installed. If I don't install the dependencies, the cherry-pick commit fails because the pre-commit hook doesn't have its dependencies. Does that make it clear why this step is necessary?

Reply:
worktree? can we remove that stuff then? should not be needed, was not part of our other cherrpick workflow, then we just executed the cherrypick command. 

What does the worktree stuff add? (I copied this from another source, so I did not write these instructions)

Reply:
@torkelo The worktree functionality is actually new to me as well, it basically creates a sandbox where you switch to a new branch, and when you're done you just exit the worktree and delete it. I don't have any opinion on it, I actually like it since it leaves your workspace clean.

Reply:
removing the worktree and replacing it with normal git switch/checkout should make sure you have your normal node_modules intact

Reply:
@torkelo OK I can make that change.

Reply:
@torkelo I removed the worktree stuff as you asked, can you have a look at the updated code please?